pmd-ui/src/main/resources/net/sourceforge/pmd/util/fxdesigner/fxml/xpath.fxml

```diff
@@ -94,7 +94,7 @@
                                 </MenuButton>
                                 <Button fx:id="exportXpathToRuleButton" mnemonicParsing="false" styleClass="icon-button">
                                     <graphic>
-                                        <FontIcon iconLiteral="fa-sign-out" />
+                                        <FontIcon iconLiteral="fa-share" />
```

Comment:
maybe using `fa-file-export` would work better?

Reply:
That would be a better idea yes, but that icon is from fontawesome 5.1.0 - and the pack we use ships version 5.0.1... Maybe I can inline the svg inside the file though

Reply:
I was able to inline the SVG. I'll open a ticket to ask them for an update though

Reply:
Note: at least on my machine the symbol is hard to read because it's too so small...

Reply:
Yes, in fact, most headers could be a bit bigger too. This will have to wait until all headers are replaced with ToolbarTitledPanes, which will allow to simplify the stylesheets and FXML. In fact atm many headers are implemented as an AnchorPane and a Toolbar right above, because I didn't know better at the time ^^

Reply:
ok, is that going to be part of this PR, or is this one feature-complete and should be merged once the current build failures are fixed?